### ISSUE TITLE
Moderation system improvements

### ISSUE THREAD

**Issue by deathsou** [Minehut/Meta#815]
### Feature request

**Checklist**
- [ ye ] I have read and agree to the README (found on https://github.com/Minehut/Meta)
- [ ye ] This is not a bug report, support request or plugin request
- [ ye ] I have searched the Issue Tracker to make sure this is not a duplicate issue
- [ ye ] This issue does not contain any sensitive information 

**Platform**
Discord, website (slightly in-game as well).

**Describe the feature**
A better moderation system and/or rule revamp. Mods on minehut have a long history of abuse; unfair mutes, bans and warnings- many of which are based on the incredibly vague rules I'll be mentioning below. 
**Feel free to disagree with some of these suggestions/listed issues, but consider each individually before disagreeing or agreeing.** 

Vague rules:
1. a general thing: comedy and satire is not taken account into the rules. a simple joke is taken seriously by mods and results in a mute. Example: arguing with someone and saying an obviously fake IP address. This is taken as a threat/leak of personal info and can lead to a mute (if a mod is present).
2. Rule no. 4. This rule is far too vague: many people are over-sensitized, and will be offended by something such as "you're short" or even the word "sword". I'd suggest keeping that rule within the limits of harassment/purposefully triggering someone. Either way, most things that would trigger someone are already banned on all platforms.
3. Rule no.5. The obvious one. 'Disrespect' is a broad term, and can range from an extreme insult to even slightly complaining about someone. The disrespect rule has been used over and over again to mute people for absolutely nothing, it's so easy to accidentally 'disrespect' someone. Suggestion: change it to insulting someone. Or something else- you're the company after all. (Also, is it disrespect to any player? I've been 'disrespected' by plenty of moderators in the past... 0_0) 
4. Rule no.8 makes sense- until you take into account the friendliness of a conversation. Two people going along well may ask what country the other lives in as part of a conversation to keep it going, or what grade they are etc. The other person might respond or not, but the one that asked will then get muted. I believe it should be changed to more specific information, since grade/country/state is not identifiable and is something anybody would like to know in a conversation.
5. Rule no.11 should take into account instigators. Someone who starts an argument will indeed lead to others arguing; but if they are muted along with the first one then I find that absurd. If I started a debate, and someone chimed in, only I should get muted, not the other. The only reason I say this is because this is how it's been used by mods in the past, the actual rule is fine, but its history isn't.

How to improve moderation system:
Bot should send a more detailed description of the mute.
This should include but is not limited to:
- An image with proof of what the mute was about
- A personalized message from the mod concerning the punishment
- Which message led to the ban, depends on situation
- (all this shouldn't apply in the case of an automatic mute due to spamming, cursing or whatnot)
There should be a command to directly appeal with the bot, such as !appeal, which then leads to the bot sending all the required information on how to appeal.
On weekends, there should be designated people to check punishment appeals, as this is when most people have time to play video games and it would suck if they couldn't appeal an unfair punishment.
Generally less bias from the mods; people with a bad rep are far more susceptible to a mute than someone with a good rep. 
Minimodding should be discouraged... not for your sake but for ours... it ruins much of the gameplay and experience in-game/discord (especially). 
Coordination between mods, so they know immediately who muted who, why, etc. Maybe also a hierarchy system where admins/sr. mods can reverse an unfair mute they see as soon as it happens. You know, like when someone on the job does a mistake and the supervisor corrects them?

**How the feature is useful**
This would greatly improve the perception of minehut from the average user, as plenty of people hate moderators and the moderation system simply because of the issues I listed above. 

-DeathSource (totally not because I was just unfairly muted)

---
**Use 👍 and 👎 reactions to indicate you agree or disagree with this suggestion! Feel free to discuss the idea in the comments.**

**Comment by honkling** [comment]
The issue with attaching an image is that it makes it far more difficult and time consuming to do so, because the bot needs to be adapted to recognize images, and the moderator would need to copy the user's id, paste it into the command in the #staff channel, go back to #general or whatever channel, find the message, take a screenshot, go back to #staff, write a reason AND now personalized message, then attach the image and send. Moderators do not have the time to do all this for every single rulebreaker. Besides, if they just attach a link to the breaking message, you can just view the link, you don't need an image.

**Comment by deathsou** [comment]
> The issue with attaching an image is that it makes it far more difficult and time consuming to do so, because the bot needs to be adapted to recognize images, and the moderator would need to copy the user's id, paste it into the command in the #staff channel, go back to #general or whatever channel, find the message, take a screenshot, go back to #staff, write a reason AND now personalized message, then attach the image and send. Moderators do not have the time to do all this for every single rulebreaker. Besides, if they just attach a link to the breaking message, you can just view the link, you don't need an image.

I understand where you're coming from; maybe an image would not be necessary. But maybe in the command you include user id + message id, the bot automatically copy pastes the message into the punishment reason? Could that work?

**Comment by honkling** [comment]
The bot would also need the channel ID, I believe.

**Comment by TrueReassembly** [comment]
Here's the issue with getting proof for every punishment, People dont understand how hard it is to moderate, the chat moves insanely fast, and if they did need to screenshot evidence of each mute, there would be way more rule breaking, Plus chat logs exist anyway.

A personal note from the mod is called the punishment reason.

!appeal does exist, you can also DM a mod to ask how to appeal.

Bias is a strong term, if you are a repeat offender, mods will be more brutal, because its clear that the user doesnt care about punishment.

Having a heirarchy for unbans / unmutes would make appeals a lot slower, since the mod who would deal with that appeal couldn't do anything

TL;DR, most points given aren't really reasonable, and is not taking into account how hard moderation is for minehut

**Comment by BennyDoesStuff** [comment]
I'm going to close this issue. As a result of the large size of Minehut, it isn't feasible for us to attach evidence and a personalized message for every punishment we issue. There's a ton of people and only a small number of mods, this would only make their jobs more difficult.

In regards to what you mentioned about the rules, we are currently in the process of updating them so that they will be more clear on exactly what we allow and don't allow, as well as where our jurisdiction exists.
